### ISSUE TITLE
Tab completion should omit " (IRC)" suffix

### ISSUE THREAD

**Issue by rhuss** [torhve/weechat-matrix-protocol-script#42]
Sorry in case this is a noob question, I'm just trying weechat and Matrix for the first time ;-)

The script very smooth, thanks a lot for your efforts !

I wonder however whether it is possible to omit the suffix when using the Matrix IRC bridge. 

Here the nick names indicate the origin (like " (IRC)") which is included when doing a tab completion to refer to somebody. This is quite confusing (as "bla (IRC)" is *not* the nickname but its "bla" for the IRC server).

would be awesome if this suffix could be avoided as e.g. in other Matric clients like Riot.

**Comment by PureTryOut** [comment]
Yes please. This already confused some people I talk with, and it's just unneeded. Both Quaternion and Riot don't do this, and it should be avoided here as well.
